drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/ScenarioRunnerServiceTest.java

```diff
@@ -65,10 +103,17 @@ public void setUp() throws Exception {
                                             projectService,
                                             scenarioLoader);
 
-        when(sessionService.newDefaultKieSessionWithPseudoClock(any(KieProject.class))).thenReturn(defaultPseudoClockKieSession);
-    }
+        final KieServices kieServices = KieServices.Factory.get();
+        final ReleaseId releaseId = kieServices.newReleaseId("org.kie.workbench.playground",
+                                                             "mortgages",
+                                                             "8.0.0-SNAPSHOT");
```

Comment:
Same question about version

Reply:
Do you know if this has to match the version in the _mortgages_ project ```pom```?

I have a feeling it could be whatever you like; and therefore values ```test```/```test```/```1.0.0-SNAPSHOT``` could be fixed (and not matter.. like a lot of the KIE CI tests do.. e.g. https://github.com/kiegroup/drools/blob/master/kie-ci/src/test/java/org/kie/scanner/KieModuleBuilderTest.java#L61)